Topic8/src/main/java/com/aguirre/topic8/services/ScheduleTimeService.java

```diff
@@ -0,0 +1,25 @@
+package com.aguirre.topic8.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.aguirre.topic8.models.ScheduleTime;
+import com.aguirre.topic8.repositories.ScheduleTimeDao;
+
+@Service
+public class ScheduleTimeService {

```

Comment:
Every Service (like the DAOs) need to have an interface and the class that implement it